lib/sandbox/vault.js

```diff
@@ -33,11 +33,11 @@ class Vault {
 const getVaultInterface = (vault) => {
     return {
         get: (key) => {
-            return vault('get', key);
+            return vault('get', key).then((value) => { return (value === null ? undefined : value); });
```

Comment:
Why is this coming `null` in the first place? This looks like a patch rather than fixing the root cause